site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -12053,6 +12053,9 @@ private void addNewSite(ParameterParser params, SessionState state) {
 					rp.addProperty(TEMPLATE_USED, templateSite.getId());
 				}
 
+				// Enable MathJax, if applies
+				MathJaxEnabler.prepareMathJaxForNewSite(site, state, (List<String>) state.getAttribute(STATE_TOOL_REGISTRATION_SELECTED_LIST));
```

Comment:
I think a more ideal solution would be to enable the site level mathjax checkbox by default, and enable users to opt-in to mathjax support as they pick their tools, like they do when enabling it manually. Having it global like this, while convenient, will likely slow down page rendering unnecessarily.